.gitignore

```diff
@@ -37,6 +37,9 @@ bower_components
 # Compiled binary addons (http://nodejs.org/api/addons.html)
 build/Release
 
+# build directory
+build/
```

Comment:
is there a `build/` directory being written in this repo? Is it just for testing?